source/2020-04-17-the-ember-times-issue-144.md

```diff
@@ -25,14 +25,19 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Quest Issue: Drop Mixins in ember-simple-auth 🔐](https://twitter.com/marcoow/status/1250411978339819522)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Marco Otte-Witte (@marcoow)](https://github.com/marcoow) announced a [quest issue to Drop Mixins](https://github.com/simplabs/ember-simple-auth/issues/2185) in [ember-simple-auth](https://github.com/simplabs/ember-simple-auth).
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The addon [ember-simple-auth](https://github.com/simplabs/ember-simple-auth) is a lightweight library for implementing authentication and authorization with Ember.js applications. When it was originally designed, the addon was heavily leveraging [mixins](https://api.emberjs.com/ember/release/classes/Mixin) so it could easily be integrated by developers into their applications. However, with [Native JavaScript classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), mixins aren't really well supported and you have to fall back to constructs like:
```

Comment:
```suggestion
The addon [ember-simple-auth](https://github.com/simplabs/ember-simple-auth) is a lightweight library for implementing authentication and authorization in Ember.js applications. When it was originally designed, the addon heavily leveraged [mixins](https://api.emberjs.com/ember/release/classes/Mixin) so that developers could easily integrate it into their applications. However, with [Native JavaScript classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), mixins aren't really well supported and you have to fall back to constructs like:
```

Reply:
Went with "one-step integrated" :)